__tests__/hooks/useGroupInfo.test.js

```diff
@@ -474,84 +478,89 @@ describe('useGroupInfo should run correctly', () => {
-      .mockResolvedValue({data: {data: {blocked: true, blocker: false}}});
-    const {result} = renderHook(() => useGroupInfo({navigation}), {wrapper: wrapper2});
-    await result.current.setSelectedUser({user_id: '123', user: {name: 'agita'}});
-    await result.current.checkUserIsBlockHandle();
-    expect(result.current.openModal).toBeFalsy();
+  // TODO: Need to fixing the test with new logic
+  // it('checkUserIsBlockHandle no block test 2 sshould run correctly', async () => {
+  //   const navigation = {
+  //     push: jest.fn(),
+  //     navigate: jest.fn(),
+  //     reset: jest.fn()
+  //   };
+  //   const spy = jest
+  //     .spyOn(serviceProfile, 'checkUserBlock')
+  //     .mockResolvedValue({data: {data: {blocked: false, blocker: false}}});
+  //   const {result} = renderHook(() => useGroupInfo({navigation}), {wrapper: wrapper2});
+  //   await result.current.setSelectedUser({user_id: '123', user: {name: 'agita', image: imageUrl}});
+  //   await result.current.checkUserIsBlockHandle();
+  //   expect(result.current.openModal).toBeFalsy();
+
+  //   expect(spy).toHaveBeenCalled();
+  //   expect(mockedPushNavigation).toHaveBeenCalled();
+  // });
 
-    expect(spy).toHaveBeenCalled();
-  });
+  // TODO: Need to fixing the test with new logic
+  // it('checkUserIsBlockHandle block sshould run correctly', async () => {
+  //   const navigation = {
+  //     push: jest.fn(),
+  //     navigate: jest.fn(),
+  //     reset: jest.fn()
+  //   };
+  //   const spy = jest
+  //     .spyOn(serviceProfile, 'checkUserBlock')
+  //     .mockResolvedValue({data: {data: {blocked: true, blocker: false}}});
+  //   const {result} = renderHook(() => useGroupInfo({navigation}), {wrapper: wrapper2});
+  //   await result.current.setSelectedUser({user_id: '123', user: {name: 'agita'}});
+  //   await result.current.checkUserIsBlockHandle();
+  //   expect(result.current.openModal).toBeFalsy();
+
+  //   expect(spy).toHaveBeenCalled();
+  // });
 
-  it('openChatMessage with no channel sshould run correctly', async () => {
-    const navigation = {
-      push: jest.fn(),
-      navigate: jest.fn(),
-      reset: jest.fn()
-    };
-    const {result} = renderHook(() => useGroupInfo({navigation}), {wrapper: wrapper2});
-    await result.current.setSelectedUser({user_id: '123', user: {name: 'agita'}});
-    await result.current.openChatMessage();
-    expect(mockedPushNavigation).toHaveBeenCalled();
-  });
+  // TODO: Need to fixing the test with new logic
+  // it('openChatMessage with no channel sshould run correctly', async () => {
+  //   const navigation = {
+  //     push: jest.fn(),
+  //     navigate: jest.fn(),
+  //     reset: jest.fn()
+  //   };
+  //   const {result} = renderHook(() => useGroupInfo({navigation}), {wrapper: wrapper2});
+  //   await result.current.setSelectedUser({user_id: '123', user: {name: 'agita'}});
+  //   await result.current.openChatMessage();
+  //   expect(mockedPushNavigation).toHaveBeenCalled();
+  // });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These tests are also commented out. Please ensure that the new logic is covered by tests.

```diff
-   // TODO: Need to fixing the test with new logic
-   // it('checkUserIsBlockHandle no block test 1 sshould run correctly', async () => {
-   //   const navigation = {
-   //     push: jest.fn(),
-   //     navigate: jest.fn(),
-   //     reset: jest.fn()
-   //   };
-   //   const spy = jest
-   //     .spyOn(serviceProfile, 'checkUserBlock')
-   //     .mockResolvedValue({data: {data: {blocked: false, blocker: false}}});
-   //   const {result} = renderHook(() => useGroupInfo({navigation}), {wrapper});
-   //   await result.current.setSelectedUser({user_id: '123'});
-   //   await result.current.checkUserIsBlockHandle();
-   //   expect(result.current.openModal).toBeFalsy();
-   //   expect(spy).toHaveBeenCalled();
-   //   expect(mockedPushNavigation).toHaveBeenCalled();
-   // });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->